setup.py

```diff
@@ -35,7 +35,12 @@ def run(cls):
     version=__version__,
 
     tests_require=test_require,
-    extras_require={"crc32c": ["crc32c"]},
+    extras_require={
+        "crc32c": ["crc32c"],
+        "lz4": ["lz4"],
+        "snappy": ["python-snappy"],
```

Comment:
I wasn't sure if we should take the `"snappy"` namespace or use `"python-snappy"` so as to disambiguate between different implementations if we later have support for a pure-python vs c-based lib... this aplpies to the `"zstandard"` one as well...

Reply:
@dpkp @tvoinarovskyi any opinion?

Reply:
I think dropping the `python-` prefix is fine here.